tests/core/gadget-config-defaults-to-snaps/task.yaml

```diff
@@ -84,6 +91,9 @@ prepare: |
     # wait for first boot to be done
     wait_for_first_boot_change
 
+    # install snap with default-configure hook
+    snap install --edge "test-snapd-with-default-configure${SUFFIX}"
```

Comment:
Thanks for adding this, it uncovered a small inconsistency - this test currently has: `systems: [ubuntu-core-1*]` which means only test-snapd-with-default-configure{,-core18} are strictly needed, the -core20/-core22 versions will never be used afaict. There is already a TODO for use to update this test to cover core20+ so probably nothing to do I was just confused for a second when reading this, hence my comment.

Reply:
Thanks, yes, indeed targets only

Reply:
As a proposal, I have removed the non core1* snaps for now, which does not currently serve a purpose and this will be added again when extending the test for core2* support.